ajax/libs/isomorphic-fetch/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "isomorphic-fetch",
+  "description": "Isomorphic WHATWG Fetch API, for Node & Browserify",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/matthew-andrews/isomorphic-fetch.git"
+  },
+  "author": "Matt Andrews <matt@mattandre.ws> (http://mattandre.ws/)",
+  "license": "MIT",
+  "filename": "fetch-npm-node.js",
+  "keywords": [
+    "fetch",
+    "api",
+    "node",
+    "browserify"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/matthew-andrews/isomorphic-fetch.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "fetch-!(npm-node)"
```

Comment:
It will still fetch `fetch-npm-node.js`. Please use `"fetch-!(npm-node)*"` instead.